projects/forge-lib/src/lib/shared/form-editor-components/concrete/currency-editor/data/currency-editor-data.component.html

```diff
@@ -0,0 +1,4 @@
+<div sws-form input-active full-width>
+  <input [(ngModel)]="this.component.data.inputMask" sws-form__input type="'text'" [name]="'mask_input'" onkeyup="this.setAttribute('value', this.value);">
+  <label sws-form__label [for]="'label_input'">Input Mask</label>
```

Comment:
language file?

Reply:
language file will be added on later phases